benchmarks/sa.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3                                                         
+
+import time, argparse
+import numpy as np
+import arkouda as ak
+
+TYPES = ('int64', 'float64', 'bool', 'str')
+
+def time_ak_sa( vsize,strlen, trials, dtype):
+    print(">>> arkouda suffix array")
+    cfg = ak.get_config()
+    Nv = vsize * cfg["numLocales"]
+    print("numLocales = {},  num of strings  = {:,}".format(cfg["numLocales"], Nv))
+    v = ak.random_strings_uniform(1, strlen, Nv)
+    c=ak.suffix_array(v)
+#    print("size of suffix array={}".format(c.bytes.size))    
+#    print("offset/number of suffix array={}".format(c.offsets.size))    
+#    print("itemsize of suffix array={}".format(c.offsets.itemsize))    
+#    print("All the random strings are as follows")
+#    for k in range(vsize):
+#       print("the {} th random tring ={}".format(k,v[k]))    
+#       print("the {} th suffix array ={}".format(k,c[k]))    
+#       print("")
+    timings = []
+    for _ in range(trials):
+        start = time.time()
+        c=ak.suffix_array(v)
+        end = time.time()
+        timings.append(end - start)
+    tavg = sum(timings) / trials
+
+    print("Average time = {:.4f} sec".format(tavg))
+    if dtype == 'str':
+        offsets_transferred = 0 * c.offsets.size * c.offsets.itemsize
+#        bytes_transferred = (c.offsets.size * c.offsets.itemsize) + (2 * c.bytes.size)
+        bytes_transferred = (c.bytes.size * c.offsets.itemsize) + (0 * c.bytes.size)
+        bytes_per_sec = (offsets_transferred + bytes_transferred) / tavg
+    else:
+        print("Wrong data type")
+    print("Average rate = {:.2f} GiB/sec".format(bytes_per_sec/2**30))
+
+def time_np_sa(Ni, Nv, trials, dtype, random):
+    print("to be done")
+
+def check_correctness(dtype, random):
+    print("to be done")
+
+def create_parser():
```

Comment:
recommend adding return type

Reply:
I do not understand how to add return type because we do not have return value.